import-service/src/handlers/importFileParser.js

```diff
@@ -0,0 +1,44 @@
+import S3 from 'aws-sdk/clients/s3';
+import { errorResponse, successResponse } from '../utils/responseBuilder';
+import winstonLogger from '../utils/logger';
+import { getProducts, moveParsedFile } from '../services/s3Service';
+
+const s3 = new S3({ region: 'eu-west-1', signatureVersion: 'v4' });
+
+exports.importFileParser = async (event) => {
+  try {
+    winstonLogger.info(
+      `Incoming request: ${JSON.stringify(event)}`,
+    );
+
+    //   const record = event.Records[0];
+    const record = {
+      s3: {
+        bucket: {
+          name: 'rs-school-import-service-bucket',
+        },
+        object: {
+          key: 'uploaded/task5_products.csv',
+        },
+      },
+    };
+    const params = {
+      Bucket: record.s3.bucket.name,
+      Key: record.s3.object.key,
+    };
```

Comment:
This is a hardcoded record. You could use for..of loop iterating over the event.Records array instead.

Reply:
Fixed in commit 30d4715